spec/support/syntax.cr

```diff
@@ -54,8 +54,12 @@ class String
     Var.new self
   end
 
-  def arg(default_value = nil, restriction = nil, external_name = nil)
-    Arg.new self, default_value: default_value, restriction: restriction, external_name: external_name
+  def ann
+    Annotation.new path
+  end
+
+  def arg(default_value = nil, restriction = nil, external_name = nil, parsed_annotations = nil)
```

Comment:
Maybe call the parameter just `annotations`?

Reply:
Wouldn't that conflict with https://github.com/crystal-lang/crystal/blob/master/src/compiler/crystal/annotatable.cr#L4?